build/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "ftlabs-screens-carousel",
+  "version": "1.0.5",
```

Comment:
Yeah there were a few test releases, I have deleted them but don't want to change history too much.